herddb-core/pom.xml

```diff
@@ -38,6 +38,11 @@
             <artifactId>herddb-utils</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>herddb-mock</artifactId>
+            <version>${project.version}</version>
```

Comment:
shouldnt it be scope=provided/optional (and only shade would be tweaked to include it)? By default grabbing core artifact I don't want it right?

Reply:
good idea

Reply:
good idea

Reply:
I am not rue about what it is better.
Currently we are not supporting Geometry and JSON functions so basically we don't need those library.
The only concern is about Applications that happen to need those jars.
In this case I guess they already have those jars on the classpath and they will have to explicitly exclude "herddb-mock".

Regarding the JDBC driver it is good to have this herddb-mock inside the JAR, properly relocated.

I am trying to add "optional" here now, and I am looking to the results